TestForSignUp/InputTextGenerator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace TestForSignUp
+{
+    internal class InputGenerator
+    {
+        internal string emailString;
+        internal string dateString;
+        internal string commonString;
+
+        internal InputGenerator()
+        {
+            emailString = Guid.NewGuid().ToString() + "@gmail.com";
+            commonString = "A";
+            foreach (var i in Guid.NewGuid().ToString()) if (char.IsLetter(i)) commonString += i;
```

Comment:
A little difficult to read, but it is just my opinion

Reply:
fixed